### PR TITLE
Opsrc 526/sylius 1.11 support

### PR DESCRIPTION
**Pull request by liszkapawel**
This PR adds support for syllius 1.11, fixes behat tests and removes scrutinizer.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -21,28 +21,22 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3, 8.0]
+                php: [8.0, 7.4]
                 symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.9.0, ~1.10.0]
-                node: [10.x]
-                mysql: [5.7]
+                sylius: [~1.9.0, ~1.10.0, ~1.11.0]
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml:26]: In `composer.json` you also support `1.8`, but there's no version in action.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -21,28 +21,22 @@ jobs:
...
+                node: [14.19]
+                mysql: [8.0]
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml]: Why? Afaik Sylius supports also MySQL 5.7 so it's safer to use the lower version.

File: composer.json [hunk 3]
```diff
@@ -39,7 +39,9 @@
         "symfony/debug-bundle": "^4.4 || ^5.2",
         "symfony/dotenv": "^4.4 || ^5.2",
         "symfony/intl": "^4.4 || ^5.2",
-        "symfony/web-profiler-bundle": "^4.4 || ^5.2"
+        "symfony/web-profiler-bundle": "^4.4 || ^5.2",
+        "friendsofsymfony/oauth-server-bundle": "2.*@dev",
```
Comment (jakubtobiasz) [inline composer.json:43]: What's the reason previously this package wasn't required and it's present?

**Review by jakubtobiasz — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (liszkapawel) [reply composer.json:43]: Without that package we can see problem like [that ](https://github.com/BitBagCommerce/SyliusMailChimpPlugin/runs/6004532776?check_suite_focus=true)

File: .github/workflows/build.yml (hunk 2 above)
Comment (liszkapawel) [reply .github/workflows/build.yml]: Reverted to 5.7

File: .github/workflows/build.yml (hunk 1 above)
Comment (liszkapawel) [reply .github/workflows/build.yml:26]: Changed

**Review by jakubtobiasz — APPROVED** [review]

**Pull request merged**